files/build_templates/host_regex.json

```diff
@@ -0,0 +1,17 @@
+[
+    {
+        "tag": "disk-usage",
```

Comment:
proc.name is monit for disk & mem usage. sshd for auth fail.
You would need to split into 2 files (monit-regex, sshd-regex) and have two entries in  the events_info.json